src/iop/basicadj.c

```diff
@@ -726,10 +726,11 @@ static inline double xlog(double d)
 
   x = x * t + 0.693147180559945286226764 * e;
 
+#if 0  // we know that xlog will only be called with values 1 <= d <= 65537
```

Comment:
Maybe remove the commented out code and replace by a longer comment saying why we are doing no further action?

Reply:
Done.  I also moved the comment to the start of the function, since the special-case checks should have been done before all of the other computation.

BTW, basicadj.c could stand getting a cleanup of the mish-mash of float/double it currently contains, in which case `xlog` will disappear entirely....